app/src/ui/clone-repository/clone-repository.tsx

```diff
@@ -503,8 +503,8 @@ export class CloneRepository extends React.Component<
             onAction={this.signInEnterprise}
           >
             <div>
-              If you have a GitHub Enterprise or AE account at work, sign in to
-              it to get access to your repositories.
+              If you are using GitHub Enterprise at work, sign in to it to get
+              access to your repositories.
```

Comment:
```suggestion
              Sign into your organization's hosted GitHub Enterprise to access your organization's repositories. 
```

Wondering if we should use some of the clarifying verbiage, you used in your PR body? Unless it is unlikely that a user would know that their GitHub is org hosted. 

Other thoughts on original verbiage of `GitHub Enterprise at work`, would lead me to assume `github.com` is for personal and `GitHub Enterprise` is for work accounts. 

Maybe `Sign in to your GitHub.com account to access your repositories` should be `Sign in to your GitHub.com (personal or work organization) account to access your repositories`

Reply:
As we discussed in our weekly we're going to align with other product's language (CLI, VsCode) for now. It's definitely still confusing but that should be at least partially mitigated by the new ability to enter github.com into the Enterprise endpoint dialog.